include/seqan3/alphabet/concept.hpp

```diff
@@ -546,7 +563,7 @@ struct assign_char_strictly_to_fn
     decltype(auto) operator()(seqan3::alphabet_char_t<alph_t> const r, alph_t & a) const
     {
         if (!seqan3::char_is_valid_for<alph_t>(r))
-            throw seqan3::invalid_char_assignment{seqan3::detail::get_display_name_v<alph_t>, r};
+            throw seqan3::invalid_char_assignment{"Alphabet", r};
```

Comment:
Mhm, this saves one include, but the error message will now be less meaningful?

Reply:
Still open, but I don't have a strong feeling about this.

Reply:
We need to do this, because `seqan3::detail::get_display_name_v<alph_t>` includes small string which in turn pulls in the complete range library. We want to avoid this.

Reply:
Sorry to unresolve this but I am unsure in how important the include graph is in contrast to readable error messages? Can we discuss this?

side note: Did you chack the error output now? I think in your case it will say "Assigning H to an alphabet of type Alphabet would incur information loss." ... which is confusing I think

Reply:
Sorry to unresolve this but I am unsure in how important the include graph is in contrast to readable error messages? Can we discuss this?

side note: Did you chack the error output now? I think in your case it will say "Assigning H to an alphabet of type Alphabet would incur information loss." ... which is confusing I think